src/actions.sol

```diff
@@ -14,145 +14,153 @@
 // You should have received a copy of the GNU Affero General Public License
```

Comment:
I wanted to touch `action.sol` only in the upcoming PR, however `forge fmt` changed the `uint`